bigbluebutton-html5/imports/api/guest-users/server/modifiers/setGuestStatus.js

```diff
@@ -22,8 +23,19 @@ export default function setGuestStatus(meetingId, intId, status, approvedBy = nu
     },
   };
 
+  /** Update position of waiting users after user has been
+   *  approved or denied by the moderator 
+   */
+  const callback = (err) => {
+    if (err) {      
+      Logger.error(`Updating position in waiting queue: ${err}`);
+    }
+    
+    updatePositionInWaitingQueue(meetingId);  
+  }
```

Comment:
Some moment in the past BBB replaced the use of callbacks in MongoDB calls as a measure of performance improvement. I believe this can be replaced with something similar you did for the `guestsWaitingForApproval` handler.